src/pages/scale.tsx

```diff
@@ -102,146 +102,137 @@ export function Scale({
             />
           </ButtonGroup>
         </Box>
-        <div
-          style={{
-            flexGrow: 1,
-            display: 'grid',
-            gridTemplateRows: 'auto 1fr',
-            gap: 8
-          }}
-        >
-          <div
-            style={{
-              display: 'flex'
+        <div style={{height: 8}}></div>
```

Comment:
Open for feedback here. We need a bit more vertical breathing room in the grid layout, opted not to adjust the `gap`, but just add some space at the top.

Reply:
Seems okay to me 👍